webapp/src/reducers/apiRequest/index.ts

```diff
@@ -0,0 +1,22 @@
+import {createSlice, PayloadAction} from '@reduxjs/toolkit';
+
+const initialState: ApiRequestCompletionState = {
+    requestes: [],
+};
+
+export const apiRequestCompletionSlice = createSlice({
+    name: 'globalModal',
```

Comment:
The name is wrong here.

Reply:
yes, this has been changed in later PR not fixing here to avoid conflicts